src/mscorlib/src/System/String.Comparison.cs

```diff
@@ -1109,7 +1122,6 @@ public Boolean StartsWith(String value, StringComparison comparisonType)
                     {
                         return false;
                     }
-
                     return (CompareInfo.CompareOrdinalIgnoreCase(this, 0, value.Length, value, 0, value.Length) == 0);
```

Comment:
Nit: You can change this to `CompareOrdinalIgnoreCase(this, value)` so that it calls the span version directly.